core/test/bmi.jl

```diff
@@ -45,7 +45,7 @@ end
 @testset "get_value_ptr" begin
     model = BMI.initialize(Ribasim.Model, toml_path)
     u0 = BMI.get_value_ptr(model, "volume")
-    @test u0 == ones(4)
+    @test collect(u0.storage) == ones(4)
```

Comment:
Hmm this seems like it should be fixed in `get_value_ptr` rather than the test. Don't we want only the storage part of u if we are asking for "volume"?